docs/explanation/cryptography.rst

```diff
@@ -0,0 +1,303 @@
+.. _explanation_cryptographic-technology:
+
+Cryptographic technology in Snapcraft
+=====================================
+
+Snapcraft uses cryptographic technologies to fetch arbitrary files over the internet,
+communicate with local processes, and store credentials. It does not directly implement
+its own cryptography, but it does depend on external libraries to do so.
+
+When building snaps, Snapcraft uses different codebases and libraries depending on
+the :ref:`base snap <base-snaps>`. This means that the cryptographic techonology used
+depends on the base snap.
+
+core24 and newer
+----------------
+
+Snaps using core24 or newer as their base snap depend on Craft Application to build. For
+more information, see the  `Craft Application cryptography`_ documentation.
+
+Downloading build dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:ref:`Plugins <plugins>` use build tools to download and verify build dependencies. Some
+plugins can provision their own build tools, while others require the build tools to be
+available on the system
+
+For more information on the use of cryptography for plugins provided by Craft Parts, see
+the `Craft Parts cryptography`_ documentation.
+
+For plugins provided by Snapcraft, the table below summarizes how plugins provision
+build tools and which build tools are used to download and verify dependencies.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Plugin
+    - Build tools used
+    - Method of provisioning the build tools
+
+  * - :ref:`Colcon <reference_colcon_plugin>`
+    - ``colcon`` and ``rosdep``
+    - not provisioned
+
+  * - Conda
+    - ``conda``
+    - Requests library and `curl`_
+
+  * - Flutter
+    - ``flutter``
+    - Git
+
+  * - :ref:`Matter SDK <reference_matter_sdk_plugin>`
+    - ``matter``
+    - Git
+
+core22
+------
+
+Snaps using core22 as their base snap depend on various craft libraries to build.
+Snapcraft's cryptographic functionality with these libraries is described below.
+
+Public key signing
+~~~~~~~~~~~~~~~~~~
+
+Snapcraft supports the adding and verification of arbitrary package repositories. For
+more information, see the `Craft Archives cryptography`_ documentation.
+
+The parts system
+~~~~~~~~~~~~~~~~
+
+Snapcraft makes use of *parts* in project files for declarative builds. Parts specified
+by the user may download arbitrary files, install packages, and more. For more
+information, see the `Craft Parts cryptography`_ documentation.
+
+Creating virtual build environments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Snapcraft instantiates and executes builds on self-allocated virtual instances. For more
+information, see `Craft Providers cryptography`_ documentation.
+
+Downloading build dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Snaps using core22 have the same plugin support as core24. See the core24 section above
```

Comment:
Let's avoid relative spatial directions for the docs themselves. They don't count for much if the reader is blind or has low vision.

```suggestion
Snaps using core22 have the same plugin support as core24. See the core24 section
```

Reply:
Go to know. https://github.com/canonical/snapcraft/pull/5478/commits/b86011f1e4714e8ec77af1eac46208a99c354aa1